.github/workflows/custom_action.yaml

```diff
@@ -14,18 +24,30 @@ jobs:
       - name: Install Terraform
         uses: hashicorp/setup-terraform@v3
         with:
-          terraform_version: "1.9.1"
+          terraform_version: "~1.10.0"
       - name: Install Task
         uses: arduino/setup-task@v2
         with:
           version: 3.x
           repo-token: ${{ secrets.GITHUB_TOKEN }}
       - name: Setup OpsLevel CLI
         uses: opslevel/actions/setup-cli@v1
-      - name: Apply
+      - name: Mask Access Token - from Github
+        if: ${{ github.event_name == 'workflow_dispatch' }}
+        run: |
+          echo "::add-mask::$API_TOKEN"
+          echo "API_TOKEN=${{ github.event.inputs.api_token }}" >> $GITHUB_ENV
+          echo "USER_EMAIL=${{ github.event.inputs.user_email }}" >> $GITHUB_ENV
+      - name: Mask Access Token - from OpsLevel
+        if: ${{ github.event_name == 'repository_dispatch' }}
+        run: |
+          echo "::add-mask::$API_TOKEN"
+          echo "API_TOKEN=${{ fromJSON(github.event.client_payload.api_token) }}" >> $GITHUB_ENV
+          echo "USER_EMAIL=${{ fromJSON(github.event.client_payload.user_email) }}" >> $GITHUB_ENV
```

Comment:
should these be the same?  - like is it completely wrong that we even use `fromJSON` ?  would it just work if we didn't?!?

Reply:
Yes and great call out! 🚀